packages/eslint-config/.eslintrc.js

```diff
@@ -71,6 +71,9 @@ module.exports = {
     "@typescript-eslint/no-unused-vars": "error",
     "@typescript-eslint/no-non-null-assertion": "error",
     "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
+    "@typescript-eslint/no-unused-expressions": ["error", { allowShortCircuit: true }],
+    "no-empty-function": "off",
+    "@typescript-eslint/no-empty-function": ["error", { "allow": ["arrowFunctions"] }],
```

Comment:
I got several empty function errors related to arrow functions in tests after updating the eslint-plugin version, so I suspect the default inspection rules changed.

I added configuration here to allow empty arrow functions.

Reply:
@ryanb-jobber do you recall where the errors were coming from? If there aren't many it might be nice to fix them as part of this…

Reply:
@seanhealy 

They were empty click handlers in 3 different test suites. I replaced them all with mock functions and that seemed to do the trick, so I removed the arrow function exception 😄